gateway/ln-gateway/src/rpc/mod.rs

```diff
@@ -229,3 +229,9 @@ pub struct FederationBalanceInfo {
     pub federation_id: FederationId,
     pub ecash_balance_msats: Amount,
 }
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct MnemonicResponse {
+    pub mnemonic: Vec<String>,
+    pub legacy_federations: Vec<FederationId>,
```

Comment:
Can you add a comment explaining what `legacy_federations` does and why it exists?

Reply:
Done